MediaBrowser.Controller/Plugins/IServerEntryPoint.cs

```diff
@@ -1,21 +1,15 @@
-using System;
 using System.Threading.Tasks;
 
 namespace MediaBrowser.Controller.Plugins
 {
     /// <summary>
     /// Interface IServerEntryPoint
```

Comment:
Again, document this to fully explain the difference between this and ILongRunningTask